CircuitPy/code.py

```diff
@@ -8,14 +8,28 @@
 import busio
 import board
 
+
 import adafruit_vl53l0x
 
 # Initialize I2C bus and sensor.
-i2c1 = busio.I2C(board.GP5, board.GP4)
-vl53_1 = adafruit_vl53l0x.VL53L0X(i2c1)
+sensor1Range = -2.0
+sensor2Range = -2.0
```

Comment:
Should these just be -2 so they don't get initialized as floating point values? If something weird happens and they fall through to the UART, it'll write a long floating point to string conversion (e.g. "-2.0000000"), won't it?

Reply:
good call, will change.